localdev/docker-compose.yaml

```diff
@@ -4,6 +4,9 @@ networks:
   kind:
     external: true
 
+volumes:
```

Comment:
Could you explain this as a comment?  

I don't really understand how this fixes the issues you have.  I'll admit that I am not familar with "docker native".

Reply:
well the volume was previously mounted from the host computer file system, macos in my case, which has performance issues on macos

i'll add a comment about what it does here